src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -34,9 +34,11 @@ public partial class CSharpCompilation
 
         private bool _needsGeneratedIsReadOnlyAttribute_Value;
         private bool _needsGeneratedIsByRefLikeAttribute_Value;
+        private bool _needsGeneratedNullableAttribute_Value;
 
         private bool _needsGeneratedIsReadOnlyAttribute_IsFrozen;
         private bool _needsGeneratedIsByRefLikeAttribute_IsFrozen;
+        private bool _needsGeneratedNullableAttribute_IsFrozen;
```

Comment:
It seems that we use the `IsFrozen` flags to enforce a phase ordering constraint. Do we really need one flag per kind of attribute, or could we do with a single one?

Reply:
It looks like we only need a single flag. Updated.